persistence-api/src/main/java/io/stargate/db/schema/Column.java

```diff
@@ -259,6 +260,18 @@ public enum Type implements ColumnType {
     Decimal(6, BigDecimal.class, false, "Variable-precision decimal, supports integers and floats"),
     Double(7, Double.class, false, "64-bit IEEE-754 floating point"),
     Duration(21, CqlDuration.class, false, "128 bit encoded duration with nanosecond precision"),
+    /**
+     * Need a placeholder for DynamicCompositeType: while not usable via APIs can potentially be
+     * created via CQL leading to problems (see {@href
+     * https://github.com/stargate/stargate/issues/2144}).
+     */
+    DynamicComposite(48, LinkedHashMap.class, false, "Legacy data type") {
```

Comment:
`LinkedHashMap` only used to be unique (not used by other types). Same true for `id`; `48` undefined.
Obviously not correct choices at this point.

Reply:
Needed to change to "real" type, can just create a placeholder similar to, say, `Point` (see below).